Adds notebook for pvcaptest

bt- opened the pull request:
@kanderso-nrel, I think I've got this all setup. Not showing a lot of engagement, which is what I expected. But, hopefully I can work on that this spring.

kandersolar commented:
Nice!  A few suggestions:
- `index.rst` needs `grid-item-card` and `toctree` entries so that this new page is listed in the index
- Cells 3 and 4 can be deleted, I think
- Cell 8, which chooses which version to use for the rest of the RTD plots, is currently using `stable`.  It might make more sense to use `latest` since it gets more traffic (presumably because it's configured as the default version in the RTD project settings).

bt- commented:
@kanderso-nrel, I added pvcaptest to the the index and cleaned up the notebook as you suggested. I also adjusted the height parameter in cell 10 to floor divide by 5 rather than 10 `height = (400 // 5) * len(subset)` because the plot came out very squished.

kanderso-nrel reviewed (approved):
Looks great, thanks!  Of course future changes/additions to make the page more useful to you are welcome as well.  

FYI right now the statistics only update when I trigger the RTD scraper (so far, once or twice a month).  It's all scripted, so it could easily be triggered more frequently, I just haven't set that up yet.
